content/src/roadmaps/tasks/understand-license-requirements-employment-agency.md

```diff
@@ -22,4 +22,4 @@ callToActionText: ""
 
 Determined the requirements for your employment and personnel services agency.
 
-:::
\ No newline at end of file
+:::
```

Comment:
Linter wants a space in it :(

Reply:
I don't see any changes made by you in this file, I recommend rolling back the changes / ignore this file from the commit.

Reply:
yes sir